pages/national_resources/gr_resources.md

```diff
@@ -1,5 +1,15 @@
 ---
 
+training:
+  - name: ELIXIR GREECE Zenodo
+    registry: Zenodo
+    description: "ELIXIR GREECE: Prospects of data and tools for ELIXIR Greece"
+    url: https://zenodo.org/records/4043630#.ZACFtE_Nx60
+  - name: ELIXIR GREECE Zenodo
```

Comment:
again name needs to be replaced with title of publication (on Zenodo) and description should be a short summary (2-3 lines on what it is about)